metagraph/docs/source/installation.rst

```diff
@@ -23,13 +23,15 @@ Docker container
 
 If docker is available on your system, you can immediately get started with::
 
-    docker run -v ${DATA_DIR_HOST}:/mnt ratschlab/metagraph \
+    docker run -v ${DATA_DIR_HOST}:/mnt ghcr.io/ratschlab/metagraph:latest \
```

Comment:
there is also one line here that probably needs to be updated
https://github.com/ratschlab/metagraph/blob/0c88d4b6c6b3128f8cc0e7036caaa89684b7f893/Dockerfile#L90

Reply:
yes good point!